### PR TITLE
Book Store: Book store components

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, I
- [ ] set up a new React application,
- [ ] Created Books, Book-List, Forms, and Categories components inside the components folder,
- [ ] Installed react-router,
- [ ] set up the router inside the App component,
- [ ] added routes for Books and categories components inside App,
- [ ] added a title and author for each book list inside the Booklist component,
- [ ] added a form for creating a new book.
- [ ] created a button inside the categories component that will check for status and,
- [ ] added a navigation component that includes links to Books and Categories components.

### CODE REVIEW

**Review by iambenkis — COMMENTED** [review]
Hi @yodit93  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that does not have a Good README template , kindly add the microverse README template 🤙 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,136 @@
...
+
+> I would like to thank [Microverseinc](https://github.com/microverseinc).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (pinkmoon25) [inline README.md:134]: Good job Implementing the project requirements, everything is working well, however I cannot yet approve the project, as currently you have a broken MIT license link present in your Readme file and in order to have professional documentation of your work you should fix this and make sure to not have any broken links in your Readme, so please fix it by adding a License file : 🙏

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @yodit93 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
Initialise Components :heavy_check_mark: 
routing :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (yodit93) [reply README.md:134]: Hello @PinkMoon25 
Thank you very much
I included the license file in the development branch by mistake. That is why the link is broken on the current branch.
I have included in this branch now.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @yodit93, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**